src/client/containers/collection/index.jsx

```diff
@@ -4,15 +4,33 @@ import UserGallery from './userGallery';
 import Projects from '../../components/projects';
 
 export default class CollectionPage extends Component {
+  generateRandomNumber() {
```

Comment:
move this function into a utility function. 

Look at https://lodash.com/docs/4.17.4#sample for inspiration

Reply:
Please delete this function if its not being used :)